### DISCUSSION TITLE
Unholy Eye

### DISCUSSION THREAD

**Discussion by TomekPL-1** [secretcowlevel/DoomtrooperPublic#4] [Card Balance]
Unholy Eye is copy paste of Evil Eye

**Comment by 21-ric** [comment]
Well, I'will keep the digital version of "unholy eye".

Unholy Eye (DARK LEGION) [C]
DARKSYMMETRY
May only be bestowed to HERETICS. Warrior gains FIRSTSTRIKE.

Original card  "evil eye" say :
DARK SYMMETRY GIFT. MAY ONLY BE BESTOWED ON HERETICS. Opponents of the Heretic are momentarily blinded. If opponent pays 3D, combat is normal. Otherwise, combat ends after Heretic attacks, and opponent may not retaliate.

I propose for this card :
DARKSYMMETRY
May only be bestowed to HERETICS. POWER IMMUNITY has no effect against this warrior. Unless your opponent pays 3[DP] during this combat. 

Now, we can talk.

**Comment by Kite80** [comment]
Italian version was very OP: the heretic gains first strike and the opponent can't retaliate. I definitely prefer the version with 3DP cost to pay for a normal fight. Losing powerimmunity because of blindness doesn't have much sense for me

**Comment by 21-ric** [comment]
@Kite80  Is there "evil eye" abily in Warzone or RPG?
Some people believe that the evil eye is a magical power to cast a spell on someone or something by looking at them, so that bad things happen to them.

**Reply by Kite80** [reply]
Don't know about warzone, but in RPG 3rd ed. there is Dark Curse which has various effects, among which the heretic can pick the blind status for his target

Dark Curse
Difficulty: D1+
Duration: Instant
Target: One living creature within Close range
Effect: This gift assails the victim with Dark Symmetry energy,
inflicting some baneful malady chosen by the Heretic. Choose one
of the following status effects: Blind, Dazed, Rattled, or Staggered.
The victim must pass a Resistance test with a difficulty equal to this
gift’s difficulty, or suffers that status effect for the next 1d6 minutes.

**Comment by 21-ric** [comment]
Evil Eye (DARK LEGION) [U]
DARKSYMMETRY
May only be bestowed to HERETICS. 
During combat, the first fate card play by your opponent has no effect.

**Reply by TomekPL-1** [reply]
This is awesome idea!

**Comment by Sol-au-fein** [comment]
DARK SYMMETRY
Opponent is blinded and suffers -2F/-2S penalty in combat

**Reply by TomekPL-1** [reply]
![insanedance](https://user-images.githubusercontent.com/83038538/116430947-958b2280-a847-11eb-873e-8e9559492cbf.JPG)

**Reply by Sol-au-fein** [reply]
Wow, what are the odds.
